src/rpc/handlers/AccountTx.cpp

```diff
@@ -180,12 +191,16 @@ doAccountTx(Context const& context)
             obj[JS(date)] = txnPlusMeta.date;
         }
         obj[JS(validated)] = true;
-
         txns.push_back(obj);
+
         if (!minReturnedIndex || txnPlusMeta.ledgerSequence < *minReturnedIndex)
+        {
             minReturnedIndex = txnPlusMeta.ledgerSequence;
+        }
         if (!maxReturnedIndex || txnPlusMeta.ledgerSequence > *maxReturnedIndex)
+        {
             maxReturnedIndex = txnPlusMeta.ledgerSequence;
+        }
```

Comment:
Can we remove the curly braces here? They are not needed, since the body is only 1 line